woocommerce/woocommerce
# Add new 2019 Algerian states

In 2019, Algerian introduced 10 new states that were previously part of other bigger states, this PR adds them

https://en.wikipedia.org/wiki/Provinces_of_Algeria#Since_2019

It also fixes some misspellings:

Closes https://github.com/woocommerce/woocommerce/issues/33969

### Testing steps:

- Make sure your shipping and billing countries include Algeria (default settings include all countries).
- In Checkout, select Algeria as a country.
- Scroll down, you should be able to see the new wilayats
<img width="497" alt="image" src="https://github.com/woocommerce/woocommerce/assets/6165348/09f8dd04-2c2a-4810-bf58-bbc51a9a6887">
